src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
@davidtheclark @jeddy3 check on `$` anywhere is bad solution, because https://www.w3.org/TR/css-variables/#defining-variables 
> A custom property is any property whose name starts with two dashes (U+002D HYPHEN-MINUS), like --foo. The <custom-property-name> production corresponds to this: it’s defined as any valid ***identifier*** that starts with two dashes. Custom properties are solely for use by authors and users; CSS will never give them a meaning beyond what is presented here.
https://www.w3.org/TR/css-syntax-3/#identifier
Ident can have escaped `$` and it is valid, try in chrome example, it is css variables, but `custom media` also is `identifier` and can have inside escaped `$`.
```css
:root {
  --color\$: red;
}

a {
  color: var(--color\$) !important;
}
```